### PR TITLE
Fix/translation-wrapping-and-consent

### PR DESCRIPTION
**Pull request by stnmtz**

### CODE REVIEW

File: src/i18n/i18n_de.properties [hunk 1]
```diff
@@ -619,26 +623,82 @@ dvo.relationship.Outgoing=Ausgehende Kontaktanfrage
...
+dvo.localRequest.status.Expired=Abgelaufen
+dvo.localRequest.status.Error=Fehlerhaft
+
+dvo.requestItem.name=Anfrage
+
+dvo.requestItem.ReadAttributeRequestItem.name=Fragte ein Attribute an
+dvo.requestItem.CreateRelationshipAttributeRequestItem.name=Erstellte ein neues Beziehungsattribut
+dvo.requestItem.CreateIdentityAttributeRequestItem.name=Erstellte ein neues Identitätsattribut
+dvo.requestItem.ProposeAttributeRequestItem.name=Schlug ein Attribut vor
+dvo.requestItem.ShareAttributeRequestItem.name=Teilte ein Attribut
```
Comment (Siolto) [inline src/i18n/i18n_de.properties:655]: Does past tense really make sense here?

File: src/i18n/i18n_de.properties [hunk 2]
```diff
@@ -810,16 +876,16 @@ dvo.attribute.name.ProprietaryBoolean=Wahrheitswert
 dvo.attribute.description.ProprietaryBoolean=Ein Wahrheitswert der WAHR oder FALSCH sein kann.
 
 dvo.attributeQuery.name.ThirdPartyRelationshipAttributeQuery=Attributsabfrage einer dritten Partei
+dvo.attributeQuery.description.ThirdPartyRelationshipAttributeQuery=This queries Relationship Attributes stored by one of your contacts.
```
Comment (Siolto) [inline src/i18n/i18n_de.properties:879]: missing translation to German

File: src/i18n/i18n_de.properties [hunk 3]
```diff
@@ -810,16 +876,16 @@ dvo.attribute.name.ProprietaryBoolean=Wahrheitswert
...
 dvo.attributeQuery.name.RelationshipAttributeQuery=Attributsabfrage einer Beziehung
+dvo.attributeQuery.description.RelationshipAttributeQuery=This queries Relationship Attributes between you and the peer.
```
Comment (Siolto) [inline src/i18n/i18n_de.properties:881]: missing translation

File: src/i18n/i18n_de.properties (hunk 1 above)
Comment (stnmtz) [reply src/i18n/i18n_de.properties:655]: Yes, as it is not decidable anymore, it is a former request thus past tense

**Review by Siolto — APPROVED** [review]

**Pull request merged**